src/static/js/modules/loan-comparison/components/loan-input-table-row.js

```diff
@@ -47,66 +12,73 @@ var LoanInputRow = React.createClass({
         loans: React.PropTypes.array.isRequired,
         scenario: React.PropTypes.object // or null
     },
-    handleChange: function (loanId, prop, eventOrVal) {
-        var val;
-        if (typeof eventOrVal === 'object' && eventOrVal) {
-            val = eventOrVal.target.value;
-        } else {
-            val = eventOrVal;
+    
+    generateClassName: function (rowType) {
```

Comment:
I'm going to start using the same naming conventions as you so we keep these functions consistent between components.